zed_components/src/zed_camera/src/zed_camera_component.cpp

```diff
@@ -3181,7 +3180,7 @@ bool ZedCamera::getCamera2BaseTransform()
     try {
         // Save the transformation
         geometry_msgs::msg::TransformStamped c2b = mTfBuffer->lookupTransform(
-            mCameraFrameId, mBaseFrameId, TIMEZERO_SYS, rclcpp::Duration(0.1));
+            mCameraFrameId, mBaseFrameId, TIMEZERO_SYS, rclcpp::Duration(0, 100000000));
```

Comment:
Why this? If a user wrongly set the TF chain `map` -> `odom` -> `base_link` -> `zed_base_link` the node will result blocked.

Reply:
in the new Duration API, 1st parameter is seconds, and the 2nd parameter is nanoseconds, so that is basically equal to what exists now.